### PR TITLE
Add workflow templates, property files, and icons

### PR DESCRIPTION
**Pull request by willbarton**
This change adds basic workflow templates for cf.gov satellite apps frontend and backend CI and releases, Python libraries, and MkDocs publishing to GitHub.

It also adds a README with some basic information about how to use the templates and what they are.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+![GitHub Actions New workflow button](github-actions-new-workflow.png)
+
+Then you will be presented with the option to create one from one of our templates, shown above.
+
+New workflow templates can be created in [github.com/cfpb/.github](https://github.com/cfpb/.github) based on [GitHub's workflow template documentation](https://docs.github.com/en/actions/configuring-and-managing-workflows/sharing-workflow-templates-within-your-organization).
+
+## consumerfinance.gov-specific workflow templates
+
+- [cfgov-frontend-ci.yml](workflow-templates/cfgov-frontend-ci.yml): A frontend CI workflow template for cf.gov satellite apps.
+- [cfgov-python-satellite-ci.yml](workflow-templates/cfgov-python-satellite-ci.yml): A backend CI workflow template for cf.gov satellite apps based on tox.
```
Comment (Scotchester) [inline README.md:20]: What would you think about reframing these as general front-end/back-end CI workflows? They offer what we would consider to be best practices for any CFPB project.

Comment (willbarton) [reply README.md:20]: We can do that...it felt somewhat presumptuous to me to do that (at least on the backend because it presupposes Python).

Comment (Scotchester) [reply README.md:20]: You can word it in such a way that it doesn't. Maybe instead of calling them frontend/backend, it should be JavaScript/Python?

Comment (willbarton) [reply README.md:20]: @Scotchester looking at this again, I'm fine moving `cfgov-frontend-ci` to `cfpb-javascript-ci`, but I think I'd like to keep the distinction between cfgov satellites and Python libraries, even if the only difference is the default of including Django in the tox matrix.

**Review by Scotchester — APPROVED** [review]

**Pull request merged**